reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -52,7 +45,22 @@ abstract class AbstractNoticeReporter : Reporter() {
             val headers: List<String>,
             val findings: LicenseFindingsMap,
             val footers: List<String>
-    )
+    ) {
+        fun cleanedFindings(copyrightGarbage: CopyrightGarbage) =
+                findings.removeGarbage(copyrightGarbage).processStatements()
+
+        private fun LicenseFindingsMap.removeGarbage(copyrightGarbage: CopyrightGarbage) =
+                mapValues { (_, copyrights) ->
+                    copyrights.filterNot {
+                        it in copyrightGarbage.items
+                    }.toMutableSet()
+                }.toSortedMap()
+
+        private fun LicenseFindingsMap.processStatements() =
```

Comment:
Nit: Maybe sort this in before `removeGarbage` to 1) have alphabetical sorting, and 2) because it "belongs" more or less to `cleanedFindings` above?

Reply:
I've added the functions in order of execution in `cleanedFindings`, seemed natural to me. But wouldn't mind changing the order.

Reply:
Ah, right, I didn't realize at first that `cleanedFindings` also calls `removeGarbage`. Then keeping the order as-is is fine with me.